src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionInitializer.java

```diff
@@ -166,4 +166,55 @@ private double estimateMinorFraction(final int segment, final AlleleFractionData
                 .mapToDouble(segment -> estimateMinorFraction(segment, data))
                 .boxed().collect(Collectors.toList()));
     }
+
```

Comment:
Just putting a note here that this block of code used to live in `AllelicPanelOfNormals` and I just moved it into this class (largely unchanged).  It will probably need updating and might get shuffled around again after #596 goes in.